app/models/draft_content_item.rb

```diff
@@ -39,6 +39,10 @@ def viewable_by?(user_uid)
     !access_limited? || authorised_user_uids.include?(user_uid)
   end
 
+  def publication_state
+    live_content_item.present? ? 'live' : 'draft'
```

Comment:
I'm going to say we should check whether there's a live item when rendering this in the presenter.